### PR TITLE
Cloud 450 prometheus alerting

### PR DESCRIPTION
**Pull request by lee-baines**

### CODE REVIEW

File: monitoring/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 The deployment uses the [CoreOS Prometheus Operator](https://coreos.com/operators/prometheus/docs/0.15.0/index.html). 
 
+Alertmanager docs: [Alertmanager](https://prometheus.io/docs/alerting/configuration/).
+
 **The monitoring folder contains the following artifacts:**
 * deploy scripts to:
     * deploy the Prometheus Operator along with Grafana and Alert Manager and other Helm charts that help monitor GKE.
```
Comment (shankar-forgerock) [inline monitoring/README.md]: s/Alert Manager/Alertmanager

Comment (shankar-forgerock) [reply monitoring/README.md]: Just an introductory note on Alertmanager would be great: The Alertmanager component of Prometheus handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct receiver integration such as email, PagerDuty, or OpsGenie. It also takes care of silencing and inhibition of alerts.

Comment (dgoldssfo) [reply monitoring/README.md]: Or, just add a second link to the Alerting concepts page: https://prometheus.io/docs/alerting/overview/

Comment (lee-baines) [reply monitoring/README.md]: I just checked and Alertmanager is a single word across all documentation. Not Alert manager

**Pull request merged**